django/db/backends/base/schema.py

```diff
@@ -1579,6 +1582,12 @@ def _rename_index_sql(self, model, old_name, new_name):
             new_name=self.quote_name(new_name),
         )
 
+    def _create_on_delete_sql(self, model, field):
+        on_delete = getattr(field.remote_field, "on_delete", None)
```

Comment:
`on_delete` is required, do we need to use `getattr()`?

Reply:
Sometimes `field.remote_field` can be `None`, getting the following error when tried without `getattr`
```
ERROR: test_remove_constraints_capital_letters (schema.tests.SchemaTests)
#23065 - Constraint names must be quoted if they contain capital letters.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.10/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.10/unittest/case.py", line 591, in run
    self._callTestMethod(testMethod)
  File "/usr/lib/python3.10/unittest/case.py", line 549, in _callTestMethod
    method()
  File "/home/akash/Djangocon/django/tests/schema/tests.py", line 4474, in test_remove_constraints_capital_letters
    "on_delete_db": editor._create_on_delete_sql(model, field),
  File "/home/akash/Djangocon/django/django/db/backends/base/schema.py", line 1590, in _create_on_delete_sql
    on_delete = field.remote_field.on_delete
AttributeError: 'NoneType' object has no attribute 'on_delete'

```